cmd/mesh-bridge/main.go

```diff
@@ -0,0 +1,112 @@
+// +build !js
+
+// mesh-bridge is a short program that bridges two Mesh nodes. This is useful in cases where
+// we introduce a network-level breaking change but still want the liquidity from one network
+//to flow to another
+package main
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/0xProject/0x-mesh/rpc"
+	"github.com/0xProject/0x-mesh/zeroex"
+	ethrpc "github.com/ethereum/go-ethereum/rpc"
+	"github.com/plaid/go-envvar/envvar"
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	firstWSRPCAddressLabel  = "FirstWSRPCAddress"
+	secondWSRPCAddressLabel = "SecondWSRPCAddress"
+	maxReceiveBatch         = 10
```

Comment:
I think it makes more sense to increase `maxReceiveBatch` and decrease `receiveTimeout`. From what we have measured so far, it looks like Mesh performs best when receiving batches of ~100 orders and usually takes less than 1 second to validate them all. How about:

```
maxReceiveBatch = 100
receiveTimeout  = 1 * time.Second
```